YPImagePickerExample/YPImagePickerExample/ExampleViewController.swift

```diff
@@ -169,7 +169,7 @@ class ExampleViewController: UIViewController {
         //
         //config.library.options = options
 
-        let picker = YPImagePicker(configuration: config)
+        let picker = YPImagePicker(configuration: config, items: selectedItems)
```

Comment:
Sorry, but this is a major update for API. Please use YPConfigLibrary to put this configuration into.